src/state.rs

```diff
@@ -146,6 +279,7 @@ impl ToSerde<SerUiState> for UiState {
     SerUiState {
       queries: queries,
       selected: self.selected,
+      colors: self.colors.unwrap_or_default(),
```

Comment:
One thing I just noticed: I think it would be great if we were to move the colors to the first position in the struct. This way, they should be the first thing in the config file as well, which makes it easier for users to find the relevant piece for modification by hand (it's much less likely a user would want to modify any of the other state by hand).

Reply:
Sure, that's easy and makes sense :)